include/certifier_tests.i

```diff
@@ -0,0 +1,26 @@
+// certifier_tests.i- SWIG interface to generate Python bindings to Certifier unit tests
+// *****************************************************************************
+// A large-ish collection of Certifier tests, defined in certifier_tests.cc and
+// related *_tests.cc files, are exposed through Swig generated Python bindings.
+// All those Certifier tests are invoked thru pytest, test_certifier_tests.py .
+// This interface file makes that glue possible through the build process(es).
+// *****************************************************************************
+
+%module libcertifier_tests
+%include "std_string.i"
```

Comment:
Deleted this line. Discovered that it's not needed in this `.i` file but IS needed in `include/certifier_framework.i`